tests/providers/test_person.py

```diff
@@ -251,6 +251,10 @@ def test_pwz_nurse(self, mock_random_digit, mock_random_int):
         assert self.factory.pwz_nurse(kind='nurse') == '4534567P'
         assert self.factory.pwz_nurse(kind='midwife') == '0317512A'
 
+    def test_nip(self):
+        for _ in range(100):
+            assert pl_validate_nip(self.factory.nip())
```

Comment:
Make sure to rebase your work against the current master. My PR #1079 was just merged, and we made some changes with variable names and stuff. Basically, use `self.fake` instead of `self.factory`.

Reply:
fix that issue, also added valid tax code offices, its long list but better to improve it